amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/AmqpChannelInitializer.java

```diff
@@ -41,6 +42,7 @@ protected void initChannel(SocketChannel ch) throws Exception {
         // octet   short      long       'size' octets   octet
 //        ch.pipeline().addLast("frameDecoder",
 //            new LengthFieldBasedFrameDecoder(MAX_FRAME_LENGTH, 3, 4, 1, 0));
+        ch.pipeline().addLast("consolidation", new FlushConsolidationHandler(1000, true));
```

Comment:
Do we need to test to find out the best value of `explicitFlushAfterFlushes `?

Reply:
Good idea, I'll add a configuration for this.